EKF/ekf.cpp

```diff
@@ -389,6 +389,8 @@ bool Ekf::initialiseFilter(void)
 		}
 	}
 
+	// warnx("_params.fusion_mode & MASK_USE_EVPOS %d", (int)(_params.fusion_mode & MASK_USE_EVPOS));
```

Comment:
I am sorry, those weren't supposed to be in. I don't have a laptop now, but I'll clean it and push tomorrow morning.